### PR TITLE
feat: Implement user and admin verification

### PR DESCRIPTION
**Pull request by amaechi-chuks**
#### Description
This PR implement user and admin verification

`Write a verification function for a user`
`Write a verification function for an admin`

#### Type of change
- [x] New feature(create a verification for user and admin)

#### How Has This Been Tested?
N/A

#### Checklist:
N/A

#### PT-ID
N/A
#### Questions:
N/A

### CODE REVIEW

File: server/helpers/validations.js [hunk 1]
```diff
@@ -17,6 +18,63 @@ const validations = {
...
     return true;
   },
+  verifyAuthHeader(req) {
+    if (!req.headers.authorization) {
+      return { error: 'error' };
+    }
+    const token = req.headers.authorization;
+    const payload = Authenticate.decode(token);
+    if (!payload) {
+      return { error: 'token' };
```
Comment (rotimi-babalola) [inline server/helpers/validations.js]: Make the error message more descriptive

File: server/helpers/validations.js [hunk 2]
```diff
@@ -17,6 +18,63 @@ const validations = {
...
+   * @method verifyAdmin
+   * @description Verifies the token provided by the Admin
+   * @param {*} req
+   * @param {*} res
+   * @param {*} next
+   * @returns {*} - JSON response object
+   */
+  verifyAdmin(req, res, next) {
+    const payload = validations.verifyAuthHeader(req);
+    const { isadmin } = payload;
```
Comment (rotimi-babalola) [inline server/helpers/validations.js]: This should be camel case

File: server/helpers/validations.js [hunk 3]
```diff
@@ -17,6 +18,63 @@ const validations = {
...
+    if (isadmin !== true) {
```
Comment (rotimi-babalola) [inline server/helpers/validations.js]: If `isAdmin` is going to be a boolean, you don't need to compare it with another boolean

File: server/helpers/validations.js [hunk 4]
```diff
@@ -17,6 +18,63 @@ const validations = {
...
+      return res.status(401).json({
+        status: 401,
```
Comment (rotimi-babalola) [inline server/helpers/validations.js]: Should be `403` which is forbidden

File: server/helpers/validations.js (hunk 1 above)
Comment (amaechi-chuks) [reply server/helpers/validations.js]: I have implemented the changes

File: server/helpers/validations.js (hunk 2 above)
Comment (amaechi-chuks) [reply server/helpers/validations.js]: I have implemented the feedback

File: server/helpers/validations.js (hunk 3 above)
Comment (amaechi-chuks) [reply server/helpers/validations.js]: I have implemented the feedback

File: server/helpers/validations.js (hunk 4 above)
Comment (amaechi-chuks) [reply server/helpers/validations.js]: I have implemented the feedback

**Pull request merged**